geonode/api/api.py

```diff
@@ -126,7 +127,14 @@ def to_json(self, data, options=None):
         counts = self.get_resources_counts(options)
         if 'objects' in data:
             for item in data['objects']:
-                item['count'] = counts.get(item['id'], 0)
+                count = counts.get(item['id'], 0)
+                if count == 0 and 'owners' in item.get('resource_uri', ''):
```

Comment:
@giohappy I had provided this check to consider the count of resources on only owners resource, what do you think?

Reply:
To be honest I don't like that hardcoded "owners" check, especially inside a generic method. Let's revert to eh previous behaviour. This PR will only reduce the returned fields but we'll keep returning users as before. We will review this with new API v2